pipelines/cv_training/cv_training_association_testing.snakefile

```diff
@@ -29,20 +24,22 @@ n_repeats = config["n_repeats"]
 debug = "--debug " if debug_flag else ""
 do_scoretest = "--do-scoretest " if config.get("do_scoretest", False) else ""
 tensor_compression_level = config["training"].get("tensor_compression_level", 1)
-model_path = Path("models")
+model_path = Path("pretrained_models")
```

Comment:
why are we calling this pre-trained models? I find it a bit confusing since its only being trained during this run and not taken from somewhere